UWPSamples/System/SystemInfoUWP/SystemInfo.cpp

```diff
@@ -419,16 +423,53 @@ void Sample::Render()
                     case GAMING_DEVICE_DEVICE_ID_XBOX_ONE_S: wcscat_s(buff, L" (Xbox One S)"); break;
                     case GAMING_DEVICE_DEVICE_ID_XBOX_ONE_X: wcscat_s(buff, L" (Xbox One X)"); break;
                     case GAMING_DEVICE_DEVICE_ID_XBOX_ONE_X_DEVKIT: wcscat_s(buff, L" (Xbox One X Dev Kit)"); break;
+
+                    #ifndef NTDDI_WIN10_NI
```

Comment:
These aren't in the Windows SDK headers until the next version of the Windows SDK, but they are already reported by the Xbox as of the April 2022 recovery.